lib/python/pyflyby/_py.py

```diff
@@ -748,8 +775,15 @@ def make_expr(arg, arg_mode=arg_mode):
                         raise _ParseInterruptedWantHelp
                     if argname in ["source"]:
                         raise _ParseInterruptedWantSource
-                if not argspec.keywords:
-                    raise ParseError("Unknown option name %s" % (argname,))
+                if sys.version_info[0] == 3:
+                    if not argspec.varkw and not argspec.kwonlyargs:
```

Comment:
We should not have the later check I guess.

```
foo(a, *, b=10, c)
```

This would not raise exception if invoked as `py --apply foo 1 --b=123 -c=1 -d=12`

Reply:
Lets add a test case for this too